lib/propolis/src/hw/virtio/softnpu.rs

```diff
@@ -110,6 +110,8 @@ pub struct SoftNpu {
 unsafe impl Send for SoftNpu {}
 unsafe impl Sync for SoftNpu {}
 
+type LoadedP4Program = (Library, Box<dyn Pipeline>);
```

Comment:
@rcgoodfellow - is this a reasonable name for this type given its contents/function? I added this to satisfy the `type_complexity` lint; let me know if you'd instead prefer for me remove this type alias and just suppress the complexity warning.

Reply:
This looks good to me.